exercises/concept-wip/crystal-hunter/.docs/instructions.md

```diff
@@ -0,0 +1,53 @@
+# Instructions
+
+You are in process of developing the new highly appreciated game **Crystal Hunter**.
+In the game you are a character that moves around and collects crystals.
+The player wins by picking up all the crystals.
+If the player comes in contact with a bandit, then the player will lose all their crystals and lose the game.
+There is an exception to this rule: the player can have an active power-up that makes them invisible to the bandits.
+
+Your goal is to write some rules that will be used in the game.
+
+## 1. Define if character gets bonus points
+
+In the game, the character will get bonus points if they touch a bandit while having a power-up.
+
+Define the `bonus_points?` method that takes two arguments (_if the character has an active power-up_ and _if the character is touching a bandit_) and returns a boolean value that tells whether the character will get bonus points or not.
+The method should return `true` only if the character has a power-up active and is touching a bandit, and `false` otherwise.
+
+```Crystal
+bonus_points?(false, true)
+# => false
+```
+
+## 2. Define if character scores
+
+In the game, the player gets points when picking up a crystal or a power-up.
+
+Define the `score?` method that takes two arguments (_if the character is touching a power-up_ and _if the character is touching a crystal_) and returns a boolean value indicating if the character scored or not.
+The method should return `true` if the character is touching a power-up or a crystal, and return `false` otherwise.
+
+```crystal
+Rules.score?(true, true)
```

Comment:
Are there power-up crystals? Shouldn't be both true I think

Reply:
There are power-ups and crystal nonrelated, you get points for picking up either.